fv3core/examples/wrapped/runfiles/baroclinic_test.py

```diff
@@ -124,7 +124,8 @@ def convert_3d_to_1d(state, field_names):
     dt_atmos = spec.namelist.dt_atmos
 
     # set backend
-    fv3core.set_backend("numpy")
+    backend = "numpy"
+    fv3core.set_backend(backend)
```

Comment:
so we are leaving the 'global_config' global state? does this still need to be set?

Reply:
Yes, it's going to be a bigger task to fully remove the global config. After this PR I'll go back to my PR running a timestep of the dynamics with no global state access, and once that test is done I can copy the pattern used to these runfiles. After that, we'll still need to refactor the unit tests to pass `Grid` explicitly to the tests instead of moving through global state. After all that, we *should* be able to remove this.

Reply:
In this specific case, we use `spec.grid` below, and `Grid` still makes use of global_config to get the backend.

Reply:
thanks!